### DISCUSSION TITLE
求助：用自编译的ISO搭建paopao网关，无法正常运行clash，一直循环尝试启动clash且无法访问clash web

### DISCUSSION THREAD

**Discussion by cliffshanger** [kkkgo/PaoPaoGateWay#21] [问和答] [answered]
INFOL0157] [505289917 3ms] inbound/proxy[tproxy-in]: inbound packet connection to 192.58.128.30:53
INFO 0157] [505289917 6ms] outbound/socks[socks-clash]: outbound packet connection to 192.58.128.30:53
ERROR[0157] [2782515060 11ms] inbound/proxyltproxy-in]: dial tcp 127.0.0.1:1080: connect: connection refused
[PaoPaoW 114432] Try to get new custom.yam]
[PaoPaoGw 114432] Load local yamI
[PaoPaoGW 114432] IOLD YAML HASH] : f427262c [PaoPaoW 1144321 ENEW YAML HASH]: £4272b2c
IPaoPaoGw 114432] Try to run Clash again .
LPaoPaoGW 1144321 Loading clash..
[PaoPaoGw 114432] ulimit adboue 1000000.
[PaoPaoGW 114432J LOKJ nft rule ICP/UDP OK.
[PaoPaoGw 1144321 EOK] SNIFF OK.
FATAL2023-09-24703:44:32.154306821 Parse config error: yam: unmarshal errors:
line 22: cannot unmarshal !!map into []map[string]interface {}
[PaoPaoGW Close Unable to close connections Delete "http://127.0.0.1:80/connections": dial tc 127.0.0.1:80: connect: connection refused %!u (MISSING)
[PaoPaoGw 114432] Same hash. Sleep 30s.
-----------------------------------------------
求助大佬帮看看是啥原因导致的，看表述像是yaml设置不对导致的，可我是按照clash.mata格式写的custom.yaml。
proxies:
    name: "xary 1.8.1"
    type: vless
    server: xx.xx.xx.xx
    port: 443
    uuid: 5fb01ea4-xxx-4013-a9d7-xxxxxxxeea066
    network: ws
    tls: true    
    udp: true
    servername: xxxxxxxxxxxxx
    skip-cert-verify: false
    fingerprint: random
    ws-opts:
    path: "/"
    headers:
    Host: xxxxxxxxxxxxxx

**Comment by kkkgo** [comment]
验证yaml是否正确的话，你可以用相关clash客户端来直接测试的，比如用clash verge之类的客户端可以直接加载测试。

**Comment by kkkgo** [comment, marked as answer]
不过很明显你这个yaml写的缩进和“-”都没有，肯定报错，可以看看官方配置文档复制照抄。
https://github.com/MetaCubeX/Clash.Meta/blob/7c59916c2220ce49be36b480797b6ffacd92e679/docs/config.yaml#L258-L487

**Reply by cliffshanger** [reply]
感谢，按照你这个YAML重新编辑了一个就好了，没想到这个YAML对格式要求有些严格。
给大佬一个赞，运行起来了，跑的很不错，丝般顺滑。

**Comment by kkkgo** [comment]
参考 https://dreamacro.github.io/clash/configuration/configuration-reference.html
参考 https://github.com/kkkgo/PaoPaoGateWay/discussions/9#discussioncomment-6207189
